distribution/docker/debian/Dockerfile

```diff
@@ -1,20 +1,7 @@
-FROM debian:buster AS builder
-
-# The `x.y.z` version of vector to fetch from packages.timber.io
-ENV VERSION nightly/latest
-
-WORKDIR /vector
-
-RUN apt-get update && apt-get install -y wget
-
-# TODO: Use https instead of http
-RUN wget https://packages.timber.io/vector/${VERSION}/vector-x86_64-unknown-linux-gnu.tar.gz
-
-RUN tar -xvf vector-x86_64-unknown-linux-gnu.tar.gz
-RUN mv vector-x86_64-unknown-linux-gnu/bin bin
-
 FROM debian:buster-slim
 
-COPY --from=builder /vector/bin/* /usr/local/bin/
+COPY vector-amd64.deb .
```

Comment:
one issue I feel with this, and is why I pulled them from s3, was that if I run this locally I need to build vector-amd64 and have all those build requirements on my system. Which I think in the case here I dont have any of the debian specific packagers. How might we be able to release this container not from CI?

Reply:
Is that a real problem? Do we need to release Docker images outside of CI? That same argument could hold true for all of the artifacts we're building (the archives, deb, rpm, etc).

Regarding pulling from S3:

1. We weren't actually pulling the right version to begin with.
2. The nightly version needs a different date-based path.
3. I think S3 is eventually consistent when changing objects?
4. We can just as easily download the archive and place it in the Docker directory if we want to release locally. This should be trivial and we can automate it in the scripts if we want.

Reply:
> We can just as easily download the archive and place it in the Docker directory if we want to release locally. This should be trivial and we can automate it in the scripts if we want.

Yeah this would work 👍 .

> I think S3 is eventually consistent when changing objects?
I believe its strongly consistent now actually, but enough to rely on it imo.

I'm happy to go with your solution. 👍